File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/xrts/XRT1002U00GrdComment3Info.java
package nta.med.data.model.ihis.xrts;

public class XRT1002U00GrdComment3Info {
	private String comments;
	private String numb;
	private String ser;
	public XRT1002U00GrdComment3Info(String comments, String numb, String ser) {
		super();
		this.comments = comments;
		this.numb = numb;
		this.ser = ser;
	}
	public String getComments() {
		return comments;
	}
	public void setComments(String comments) {
		this.comments = comments;
	}
	public String getNumb() {
		return numb;
	}
	public void setNumb(String numb) {
		this.numb = numb;
	}
	public String getSer() {
		return ser;
	}
	public void setSer(String ser) {
		this.ser = ser;
	}   

}

File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Xrts/XRT1002U00GrdComment3Info.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Xrts
{
    [Serializable]
    public class XRT1002U00GrdComment3Info
    {
        private String _comments;
        private String _numb;
        private String _ser;

        public String Comments
        {
            get { return this._comments; }
            set { this._comments = value; }
        }

        public String Numb
        {
            get { return this._numb; }
            set { this._numb = value; }
        }

        public String Ser
        {
            get { return this._ser; }
            set { this._ser = value; }
        }

        public XRT1002U00GrdComment3Info() { }

        public XRT1002U00GrdComment3Info(String comments, String numb, String ser)
        {
            this._comments = comments;
            this._numb = numb;
            this._ser = ser;
        }

    }
}